Source/Fuse.Controls.ScrollView/ScrollView.Layout.uno

```diff
@@ -230,5 +230,46 @@ namespace Fuse.Controls
 			return 0;
 		}
 		
+		/**
+			Used to specify the target for measuring distance from in `DistanceFromView` method. Can be either `Start` or `End`.
+		*/
+		internal enum DistanceFromViewTarget
+		{
+			/**
+				When used in `DistanceFromView` method, measures the distance from the view start.
+			*/
+			Start,
+			/**
+				When used in `DistanceFromView` method, measures the distance from the view end.
+			*/
+			End
+		}
+		/**
+			The distance from the visible view area start or end for the provided rectangle. It will be zero if any part of it resides before or after the view area start, respectively.
+		*/
+		internal float2 DistanceFromView(float2 min, float2 max, DistanceFromViewTarget target)
```

Comment:
Hmm, something still doesn't feel right here. You only use one of `min` or `max`. It should probably be just one parameter called `position`. You'll find you cold probably simplify the code if you do this.

Can the `DistanceToViewLinear` also be merged into this function